src/components/Search/Search.scss

```diff
@@ -10,7 +10,7 @@
   .wkp-search-button {
     border: none;
     color: white;
-    height: 41px;
+    height: 39px;
```

Comment:
breaks on large screen the button doesn't touch the bottom border

Reply:
fixed